CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -321,8 +321,8 @@
         <note/>
       </trans-unit>
       <trans-unit id="Each TV show can have their own settings, set by tapping the `Hide Spoilers…` button when looking at a show or episode." xml:space="preserve">
-        <source>Each TV show can have their own settings, set by tapping the `Hide Spoilers…` button when looking at a show or episode.</source><target>Für individuelle Einstellungen pro TV-Sendung kannst du auf `Spoiler ausblenden` tippen, während du dir die Details einer Sendung oder Episode anschaust.</target>
-        <note/>
+        <source>Each TV show can have their own settings, set by tapping the `Hide Spoilers…` button when looking at a show or episode.</source><target state="needs-review-translation">Für individuelle Einstellungen pro TV-Sendung kannst du auf `Spoiler` `ausblenden` tippen, während du dir die Details einer Sendung oder Episode anschaust.</target>
```

Comment:
The wide space here really doesn't look like it should be like that ;)

![IMG_7322](https://github.com/cliss/callsheet-localizations/assets/18497564/634101e1-6507-413a-9af8-919f486bc819)

Reply:
Wait, which space?

Also, unrelated, looks like I never added the text for those toggles to the list of items to be translated. (`Hide character names`, `Hide character episode counts`, etc). Will do so in a few.

Reply:
The full-width between "Spoiler" and "ausblenden". It's technically "correct", but looks like shitty kerning ;)

Reply:
Oh, yeah, it’s because I monospaced that. It’s the same in English IIRC.

Reply:
You could try it in English with the backtick-word-backtick-space-backtick-word-backtick hack and see if it looks more pleasing to you.

Reply:
(Had to write it like that as f*** me if I know how to escape all these backticks here)

Reply:
Damn, you're right. This is such an obvious hack that did not even occur to me. 🤦 

<img width="408" alt="Screenshot 2024-05-13 at 4 05 17 PM" src="https://github.com/cliss/callsheet-localizations/assets/282460/fd96bc0a-b359-461e-baf4-7d9a4f220c5f">

I owe you a 🍺